mappings/net/minecraft/util/dynamic/Codecs.mapping

```diff
@@ -26,10 +26,27 @@ CLASS net/minecraft/class_5699 net/minecraft/util/dynamic/Codecs
 		ARG 1 element
 	METHOD method_32852 (Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Z
 		ARG 1 element
+	METHOD method_33817 either (Lcom/mojang/serialization/Codec;Lcom/mojang/serialization/Codec;)Lcom/mojang/serialization/Codec;
```

Comment:
Mind see #2084?

Reply:
Sure, will use that instead

Reply:
In fact if yours is merged first I can fix conflicts. Not a big problem